astropy/timeseries/tests/test_binned.py

```diff
@@ -93,201 +110,374 @@ def test_initialization_n_bins_invalid_arguments():
     # any of the parameters 'time_bin_start' or 'time_bin_end' is not scalar.
 
     with pytest.raises(TypeError) as exc:
-        BinnedTimeSeries(time_bin_start=Time([1, 2, 3], format='cxcsec'),
-                         time_bin_size=1*u.s,
-                         n_bins=10)
-    assert exc.value.args[0] == ("'n_bins' cannot be specified if 'time_bin_start' or "
-                                 "'time_bin_size' are not scalar'")
+        BinnedTimeSeries(
+            time_bin_start=Time([1, 2, 3], format='cxcsec'),
+            time_bin_size=1 * u.s,
+            n_bins=10,
+        )
+    assert exc.value.args[0] == (
+        "'n_bins' cannot be specified if 'time_bin_start' or "
+        "'time_bin_size' are not scalar'"
+    )
 
 
 def test_initialization_n_bins():
 
     # Make sure things crash with incorrect n_bins
 
     with pytest.raises(TypeError) as exc:
-        BinnedTimeSeries(data={"time": ["2016-03-22T12:30:31"]},
-                         time_bin_start=Time(1, format='mjd'),
-                         time_bin_size=1*u.s,
-                         time_bin_end=Time(1, format='mjd'),
-                         n_bins=10)
-    assert exc.value.args[0] == ("'n_bins' has been given and it is not the "
-                                 "same length as the input data.")
+        BinnedTimeSeries(
+            data={"time": ["2016-03-22T12:30:31"]},
+            time_bin_start=Time(1, format='mjd'),
+            time_bin_size=1 * u.s,
+            time_bin_end=Time(1, format='mjd'),
+            n_bins=10,
+        )
+    assert exc.value.args[0] == (
+        "'n_bins' has been given and it is not the " "same length as the input data."
+    )
 
 
 def test_initialization_non_scalar_time():
 
     # Make sure things crash with incorrect size of time_bin_start
 
     with pytest.raises(ValueError) as exc:
-        BinnedTimeSeries(data={"time": ["2016-03-22T12:30:31"]},
-                         time_bin_start=["2016-03-22T12:30:31", "2016-03-22T12:30:32"],
-                         time_bin_size=1*u.s,
-                         time_bin_end=Time(1, format='mjd'))
-    assert exc.value.args[0] == ("Length of 'time_bin_start' (2) should match table length (1)")
+        BinnedTimeSeries(
+            data={"time": ["2016-03-22T12:30:31"]},
+            time_bin_start=["2016-03-22T12:30:31", "2016-03-22T12:30:32"],
+            time_bin_size=1 * u.s,
+            time_bin_end=Time(1, format='mjd'),
+        )
+    assert exc.value.args[0] == (
+        "Length of 'time_bin_start' (2) should match table length (1)"
+    )
```

Comment:
As in the other PR I reviewed, I noticed we have a lot of old-style exception checking, so we should really clean this up to use ``match=`` instead - but doesn't have to be done as part of this PR.